src/pudl/transform/ferc1.py

```diff
@@ -672,16 +672,22 @@ def aggregate_duplicate_fuel_types_xbrl(
             columns=["fuel_units_count"]
         )
 
-    def drop_null_data_rows(self, df: pd.DataFrame) -> pd.DataFrame:
-        """Drop rows in which all data columns are null.
+    def drop_total_rows(self, df: pd.DataFrame) -> pd.DataFrame:
+        """Drop rows that represent plant totals rather than individual fuels.
+
+        This is an imperfect, heuristic process. The rows we identify as probably
+        representing totals rather than individual fuels:
+
+        * have zero or null values in all of their numerical data columns
+        * have no identifiable fuel type
+        * have no identifiable fuel units
+        * DO report a value for MMBTU / MWh (heat rate)
 
         In the case of the fuel_ferc1 table, we drop any row where all the data columns
         are null AND there's a non-null value in the ``fuel_mmbtu_per_mwh`` column, as
         it typically indicates a "total" row for a plant. We also require a null value
         for the fuel_units and an "other" value for the fuel type.
 
-        Right now this is extremely stringent and almost all rows are retained.
```

Comment:
why remove this? is this no longer correct? I think this is a helpful lil tidbit.

Reply:
I made this comment in relation to the idea that this was supposed to be dropping null / invalid records **in addition to** dropping total rows, but that wasn't really what the function was doing before -- it was **only** dropping total rows, and keeping about 100,000 functionally null records around.

Now that this is method is just a minor add-on / refinement that comes after the much more sweeping `drop_invalid_rows()` it didn't seem as weird/noteworthy that it only drops a small number of rows.